analyze.info.yml

```diff
@@ -1,7 +1,6 @@
 name: 'Analyze'
 type: module
 description: 'API for adding information in the Analyze tab of entities'
-core_version_requirement: ^10
-package: Custom
-dependencies:
-  - drupal:node
+core_version_requirement: ^10 | ^11
```

Comment:
We need to up the minimum req. to 10.3 because we use single directory components